go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/coreos/etcd v3.3.18+incompatible
 	github.com/gobwas/glob v0.2.3
 	github.com/gogo/protobuf v1.3.1
-	github.com/golang/protobuf v1.3.2
+	github.com/golang/protobuf v1.4.2
```

Comment:
We found that protobuf 1.4.x has performance problems and we decided to stay with 1.3.5 because of that.